samples/msal-browser-samples/TestingSample/playwright.config.ts

```diff
@@ -0,0 +1,43 @@
+import { defineConfig, devices } from "@playwright/test";
+
+export default defineConfig({
+    testDir: "./test",
+    timeout: 30 * 1000,
+    expect: {
+        /**
```

Comment:
instead of this comment here, can we just give a pointer to the playwright configuration docs somewhere above?

Reply:
Done!